src/payment/checkout/Checkout.jsx

```diff
@@ -64,15 +72,69 @@ class Checkout extends React.Component {
     );
   };
 
+  handleSubmitStripe = () => {
+    // TODO: We'll want to submit formData here in the next iteration
+    console.log('[Project Zebra] handleSubmitStripe called');
+    this.props.submitPayment({ method: 'stripe' });
+  };
+
+  handleSubmitStripeButtonClick = () => {
+    console.log('[Project Zebra] handleSubmitStripeButtonClick');
+    sendTrackEvent(
+      'edx.bi.ecommerce.basket.payment_selected',
+      {
+        type: 'click',
+        category: 'checkout',
+        paymentMethod: 'Credit Card - Stripe',
+        checkoutType: 'client_side',
+      },
+    );
+  };
+
   handleSubmitFreeCheckout = () => {
     sendTrackEvent(
       'edx.bi.ecommerce.basket.free_checkout',
       { type: 'click', category: 'checkout' },
     );
   };
 
+  renderBillingFormSkeleton() {
+    return (
+      <>
```

Comment:
Is this a shell for tabular data?  If so it might need some ARIA roles.

Reply:
@wittjeff that's a React Fragment, it will not be rendered in the DOM.